.travis.yml

```diff
@@ -1,25 +1,22 @@
 language: rust
-osx_image: xcode8.2
+osx_image: xcode9.2
 
-rust:
-  - stable
-  - nightly
+git:
+  submodules: true
 
 os:
-  - linux
   - osx
 
-matrix:
-  allow_failures:
-    - rust: nightly
+rust:
+  - 1.29.2
+
+cache: cargo
 
-cache:
-  directories:
-    - $HOME/.cargo
+before_script:
+  - sudo gem install xcpretty
```

Comment:
I suspect we need `brew install cmake`. This used to be in the readme but has gone missing.

Reply:
nevermind I guess? I thought travis was going to fail because I saw an error in the report, seems fine though

Reply:
adding